examples/applications/plot_stock_market.py

```diff
@@ -170,7 +170,7 @@
 from sklearn import manifold
 
 node_position_model = manifold.LocallyLinearEmbedding(
-    n_components=2, eigen_solver="dense", n_neighbors=6
+    n_components=2, eigen_solver="dense", n_neighbors=6, random_state=42
```

Comment:
We don't need a random state here because the solver is `"dense"` and we only need the random state for `"arpack"`.

We can therefore let it as-is.

```suggestion
    n_components=2, eigen_solver="dense", n_neighbors=6
```